scripts/readmeFiles.js

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env node
+const fs = require("fs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const shell = require("shelljs");
+// eslint-disable-next-line import/no-extraneous-dependencies
+const reactDocs = require("react-docgen");
+
+const skipPackages = ["helpers", "Stylers", "Tokens", "Guard", "Icon", "Overlay", "Select"];
+
+const githubIssueBody = encodeURIComponent(`
+# Help wanted
+
+## Please write your question.
+*A clear and concise description of what the question is*
+
+## Additional context
+*Add any other context or screenshots about your question here.*
+`);
+
+const renderReadmeTemplate = ({
+  displayName = "",
+  name = "",
+  description = "** require description **",
+  content = "** require content **",
+  props = "",
+  version = "",
+}) => {
+  return `
+<!-- autogenerated don't modify -->
+
+# ${name} - ${version}
+
+## Description
+${description}
+
+## Installation
+
+\`\`\`
+yarn add ${name}
+\`\`\`
+
+or with npm:
+
+\`\`\`
+npm install ${name}
+\`\`\`
+
+## Props 
+${props}
+
+<!-- autogenerated don't modify -->
+${content}
+
+## Links
```

Comment:
can we actually put the links near the top, i think these are quite important, sucks to scroll all the way to the bottom for the links